configs/15.0/ubuntu/Dockerfile-devel_ppc64le

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG OS_NAME=ubuntu
+ARG OS_RELEASE=focal
+ARG AT_VERSION=15.0
+
+FROM ppc64le/${OS_NAME}:${OS_RELEASE}
+ARG OS_NAME
+ARG OS_RELEASE
+ARG AT_VERSION
+ARG REPO
+
+WORKDIR /root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN [ -n "$HTTP_PROXY" ] && echo Acquire::http::proxy '"'$HTTP_PROXY'";' >> /etc/apt/apt.conf.d/http_proxy || :
+RUN [ -n "$HTTPS_PROXY" ] && echo Acquire::https::proxy '"'$HTTPS_PROXY'";' >> /etc/apt/apt.conf.d/https_proxy || :
+RUN [ -n "$FTP_PROXY" ] && echo Acquire::ftp::proxy '"'$FTP_PROXY'";' >> /etc/apt/apt.conf.d/ftp_proxy || :
+
+RUN apt update \
+    && apt install -y apt-utils curl gnupg2 \
+        autoconf automake bison byacc ccache cscope ctags flex elfutils libtool m4 make man-db \
+    && apt clean
+
+ENV http_proxy $HTTP_PROXY
+
+RUN curl -s http://public.dhe.ibm.com/software/server/POWER/Linux/toolchain/at/${OS_NAME}/dists/${OS_RELEASE}/6976a827.gpg.key | apt-key add - \
+    && echo "deb ${REPO}/${OS_NAME} ${OS_RELEASE} at${AT_VERSION}" >> /etc/apt/sources.list \
+    && apt update \
+    && apt install -y advance-toolchain-at${AT_VERSION}-runtime \
+        advance-toolchain-at${AT_VERSION}-libnxz \
+        advance-toolchain-at${AT_VERSION}-mcore-libs \
+        advance-toolchain-at${AT_VERSION}-devel \
+        advance-toolchain-at${AT_VERSION}-perf
```

Comment:
I think we need another variable for these lines so that we can build non-GA containers, e.g. `15.0-beta1`.
Meanwhile, the repository still depends on `at15.0`.

Reply:
Added.
I've modified a couple of variable names. I've also tried to simplify the directories and files.